### PR TITLE
Style cleanup for `models/ed/R/*.R` scripts

### PR DESCRIPTION
**Pull request by bpbond**
Ran through formatR::tidy_dir with arrow=TRUE and indent=2; made sure all if blocks have braces; took advantage of paste0 and file.path; changed require calls to library. Tried to make sure `formatR` didn’t make things too jaggy. Moved ellipses to end of met2model definition.

### CODE REVIEW

**Comment by mdietze** [comment]
@bpbond ED looks good, just needs "Update branch"

File: models/ed/R/met2model.ED2.R [hunk 1]
```diff
@@ -30,297 +30,299 @@
...
-    logger.warn("Longitude does not match that of file", lon, "!=", flon)
+  dir.create(met_folder, recursive = TRUE, showWarnings = FALSE)
+  
+  ### FUNCTIONS
+  dm <- c(0, 32, 60, 91, 121, 152, 182, 213, 244, 274, 305, 335, 366)
+  dl <- c(0, 32, 61, 92, 122, 153, 183, 214, 245, 275, 306, 336, 367)
+  month <- c("JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC")
+  mon_num <- c("01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12")
+  day2mo <- function(year, day) {
+    leap      <- (year%%4 == 0)
```
Comment (dlebauer) [inline models/ed/R/met2model.ED2.R]: This is incorrect use `leap_year`

File: models/ed/R/met2model.ED2.R [hunk 2]
```diff
@@ -30,297 +30,299 @@
...
+    skip <- FALSE
+    nyr  <- floor(length(sec) / 86400 / 365 * dt)
+    yr   <- NULL
+    doy  <- NULL
+    hr   <- NULL
+    asec <- sec
+    for (y in (year + 1):(nyr - 1)) {
+      ytmp <- rep(y, 365 * 86400 / dt)
+      dtmp <- rep(1:365, each = 86400 / dt)
+      if (y%%4 == 0) {
```
Comment (dlebauer) [inline models/ed/R/met2model.ED2.R]: Here too

File: models/ed/R/met2model.ED2.R (hunk 1 above)
Comment (bpbond) [inline models/ed/R/met2model.ED2.R]: Wow, yes.

File: models/ed/R/met2model.ED2.R (hunk 2 above)
Comment (bpbond) [inline models/ed/R/met2model.ED2.R]: To fix this - can I go through the `models` scripts and have them all use `lubridate::leap_year`?

Comment (mdietze) [inline models/ed/R/met2model.ED2.R]: Will need to be a new PR

Comment (bpbond) [inline models/ed/R/met2model.ED2.R]: Agreed. I'll look around. Thanks.

**Pull request merged**